components/templates/Personalizer.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import Head from 'next/head';
+import ReactMarkdown from 'react-markdown';
+import moment from 'moment';
+import { Oval } from 'svg-loaders-react'
+import { getRecommendations } from '../helpers/api'
+import CustomList from "../molecules/CustomList";
+import DropDown from '../molecules/DropDown';
+import RadioButtons from '../molecules/RadioButtons';
+import Card from "../organisms/Card";
+import AdditionalLink from '../atoms/AdditionalLink';
+import Provinces from '../../data/provinces.json';
+import Months from '../../data/months.json';
+import styles from '../../styles/Personalizer.module.css';
+
+export default function Personalizer(props) {
+
+    const [province, setProvince] = useState(props.t.locale === "en" ? "Ontario" : "Quebec");
```

Comment:
NOT-BLOCKING: I'm sure you know this isn't how location works... :)

Reply:
I brought up with @wolfewylie that we'd need some sort of geolocation API in order to turn coordinates into a province. He suggested for the proof of concept to default to Ontario if English and Quebec if French.